ft/loader/loader.cc

```diff
@@ -719,31 +722,23 @@ static int bl_finish_compressed_write(FILE *stream, struct wbuf *wb) {
     // Mark as written
     wb->ndone = 0;
 
-    size_t size_to_write = total_size + 4; // Includes writing total_size
+    size_t size_to_write = total_size + 4;  // Includes writing total_size
+
+    r = do_fwrite(compressed_buf, 1, size_to_write, stream);
 
-    {
-        size_t written = do_fwrite(compressed_buf, 1, size_to_write, stream);
```

Comment:
See toku_os_fwrite()==toku_os_fwrite_with_source_location(), and the latter does the same, so there is no need to repeat the code here.